docs/RELEASE.md

```diff
@@ -67,7 +80,13 @@ Publishing involves creating a release tag and creating a new Release on GitHub.
     git push <YOUR FORK>
     ```
 
-1. Once the PR is merged to master, tag master with release version and push tags to remote repository.
+1. Once the PR is merged to `master`, cherry pick the commit to the `release-X.X` branch:
```

Comment:
open PR to master and a cherry-pick PR to `release-X.X`

Reply:
cherry pick commands:
```bash
export GITHUB_USER=<user name>
hack/cherry_pick_pull.sh upstream/<release branch> <pr number>
```